### PR TITLE
Add missing python dependencies to kirkstone

### PR DESCRIPTION
**Pull request by hikinggrass**

### CODE REVIEW

**Review by sdrabb — APPROVED** [review]
working for me

File: recipes-core/josev/python3-iso15118_2023.7.0.bb [hunk 1]
```diff
@@ -16,6 +16,14 @@ RDEPENDS:${PN} += " \
...
     ${PYTHON_PN}-py4j \
+    ${PYTHON_PN}-aiofile \
+    ${PYTHON_PN}-environs \
+    ${PYTHON_PN}-marshmallow \
+    ${PYTHON_PN}-netifaces \
+    ${PYTHON_PN}-python-dotenv \
+    ${PYTHON_PN}-typing-extensions \
+    ${PYTHON_PN}-caio \
+    ${PYTHON_PN}-packaging \
 "
```
Comment (sdrabb) [inline recipes-core/josev/python3-iso15118_2023.7.0.bb]: in order to be able to join nto the system would be nice to have `debug-tweaks`

**Review by sdrabb — APPROVED** [review]

File: recipes-core/josev/python3-iso15118_2023.7.0.bb (hunk 1 above)
Comment (hikinggrass) [reply recipes-core/josev/python3-iso15118_2023.7.0.bb]: isn't that typically something you specify in your local.conf?
Simply adding
```bitbake
EXTRA_IMAGE_FEATURES += "debug-tweaks"
```
There should be enough

Comment (sdrabb) [reply recipes-core/josev/python3-iso15118_2023.7.0.bb]: 👍 yup understood : )

**Review by a-w50 — APPROVED** [review]
:1st_place_medal:

**Pull request merged**